tbprofiler/6.2.1/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM mambaorg/micromamba:1.5.8 as app
+
+USER root
+WORKDIR /
+
+ARG TBPROFILER_VER="6.2.1"
+
+# this version is the shortened commit hash on the `master` branch here https://github.com/jodyphelan/tbdb/
+# commits are found on https://github.com/jodyphelan/tbdb/commits/master
+# this was the latest commit as of 2024-05-10
+ARG TBDB_COMMIT="97b5876a1a9d2f06e8bb6076ed88baf481ef5fe5"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+LABEL base.image="micromamba:1.5.8"
+LABEL dockerfile.version="1"
+LABEL software="tbprofiler"
+LABEL software.version="${TBPROFILER_VER}"
+LABEL description="The pipeline aligns reads to the H37Rv reference using bowtie2, BWA or minimap2 and then calls variants using bcftools. These variants are then compared to a drug-resistance database."
+LABEL website="https://github.com/jodyphelan/TBProfiler/"
+LABEL license="https://github.com/jodyphelan/TBProfiler/blob/master/LICENSE"
+LABEL maintainer="John Arnn"
+LABEL maintainer.email="jarnn@utah.gov"
+LABEL maintainer2="Curtis Kapsak"
+LABEL maintainer2.email="kapsakcj@gmail.com"
```

Comment:
Did you want to add yourself as a maintainer?

Reply:
I did, thank you!